### ISSUE TITLE
After running every Script i get System Error in Privacy Settings

### ISSUE THREAD

**Issue by Supremerino** [W4RH4WK/Debloat-Windows-10#254]
After running every Script, i get an System Error in Privacy Settings/App Permissions and i can't change anything. 
I'm trying to change the App permission in the privacy settings.
And instantly when i hit anything in the App permission section (for example Microphone or Calender) the Error appears. 

Complete Error: systemsettings.exe - System Error
"The system detected an overrun of a stack-based buffer in this application. This overrun could potentially allow a malicious user to gain control of this application"

Screenshots included. I marked the Problem Section Yellow.  All Other Options in Privacy Settings do work. (Windows Perm.)
![SYKIUXr](https://user-images.githubusercontent.com/65544543/86531108-c8ca8280-bebe-11ea-836d-27fd528fb7d7.png)
![HRrn8bs](https://user-images.githubusercontent.com/65544543/86531110-cb2cdc80-bebe-11ea-8b49-e358e739ce9b.png)

**Comment by W4RH4WK** [comment]
Thank you for reporting this. This seems to be the first incident of such an issue.

My first guess is that this is caused by disabling services. Can you try re-enabling services disabled by your script set.

Judging by the error message, there also seems to be a bug with the System Settings app. Of course it could be entirely possible that the bug is triggered by the scripts, yet, given it's a bug, it may be fixed in a future Windows update.

**Comment by dawidsekula** [comment]
I confirm. 'Network & Internet' is also unaccessable with the same error pop-up and 'Bluetooth & other devices' closes window immediately without error displayed.

**Comment by Silential** [comment]
I did a bit of tinkering with the scripts and restore points. whatever the issue is, its definitely caused by something in the disable-services.ps1 script.

that being said i don't know which services are causing the issue, I didn't have the time to look through all of them.

**Comment by Luffitys** [comment]
@Silential I'm on an Insider Build and the service 'wscsvc' can not be disabled. Could you try to enable that particular service and test if the issue is resolved?

**Comment by Silential** [comment]
Sorry, I totally forgot about this thread, but yes, enabling 'wscsvc' does indeed fix the issue.

**Comment by Luffitys** [comment]
#262

**Comment by pradeep-shanmugham** [comment]
I've got the same issue on version 1909. My wscsvc is running though - still get the same error and for the same set of App permissions highlighted in yellow by @Supremerino 

Are there any other services that also need to be running?

**Comment by W4RH4WK** [comment]
I don't know off the top of my head. And getting a 1909 installation of Windows just to try it out might be a bit hard. My suggestion would be upgrade your Windows. Chances are, it will restore the activation state of the services. You can then run the scripts again after upgrading. If the problem persists, it's something we have to look into.

**Comment by Supremerino** [comment]
Hello, I'm getting back to you after a long time.

I had actually solved the problem.
I can't say if it's the same problem for you, but a script had edited the "Local Computer Policies" for me. 

To fix it I went into the "Local Group Policy Editor" > Computer Configuration > Administrative Templates > Windows Components > App Privacy > then set everything that doesn't work to Enabled
I hope this helps you guys too.
